lib/gui/components/progress-button/templates/progress-button.tpl.html

```diff
@@ -3,5 +3,5 @@
     'progress-button--striped': striped && striped != 'false'
   }">
     <span class="progress-button__content" ng-transclude></span>
-    <span class="progress-button__bar" ng-style="{ width: Math.min(100, percentage) + '%' }"></span>
+    <span class="progress-button__bar" ng-style="{ width: progress.min(100, percentage) + '%' }"></span>
```

Comment:
Maybe worth just replicating `Math.min` with a ternary conditional operator, and avoid the controller altogether?

```js
percentage > 100 ? 100 : percentage
```